reporting-grofwild/inst/ui/ui.R

```diff
@@ -53,8 +53,8 @@ shinyUI(
                 tags$p("Bovenaan deze pagina krijgt u, in functie van de gekozen jachtwildsoort, de regio-schaal en de gewenste jaren en periodes,", tags$br(),"een kaart met de geografische spreiding van het afschot en een figuur met de evolutie van het afschot de laatste jaren.",
                     "Daaronder kunnen verdere grafieken worden aangeklikt, die telkens kunnen worden aangepast naargelang de informatie die u zelf wenst te visualiseren."),
 
-                tags$p(paste0("Onderstaande tabellen en figuren zijn gebaseerd op de beschikbare gegevens op ", format(as.Date(attr(ecoData, "Date")), "%d/%m/%Y"), "."))
-
+                tags$p("Onderstaande tabellen en figuren zijn gebaseerd op de beschikbare gegevens in januari 2019. Een deel data van het laatste kwartaal van 2018 kunnen mogelijks dus nog niet opgenomen zijn in de dataset",tags$br(),"Indien u fouten zou ontdekken of merkt dat data ontbreken gelieve dit dan te melden via een email naar faunabeheer@inbo.be"),
```

Comment:
En ga je dat dan elke 3 maand ook consequent updaten en dat nooit vergeten?

Reply:
is het niet beter hier iets toe te voegen in de aard van format(max(ecoData$afschot_datum, na.rm = T), "%d/%m/%Y")?

zo geef je steeds de meest recente datum in de data weer.